dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/persistence/ConsentAwareStorage.kt

```diff
@@ -63,7 +63,7 @@ internal class ConsentAwareStorage(
             callerClass = ConsentAwareStorage::class.java.name,
             metric = TelemetryMetricType.MethodCalled,
             samplingRate = MethodCallSamplingRate.RARE.rate,
-            operationName = "writeCurrentBatch[${orchestrator?.getRootDir()?.nameWithoutExtension}]"
+            operationName = "writeCurrentBatch[${orchestrator?.getRooDirName()}]"
```

Comment:
Side note: I'm not even sure what is the point of having directory name here. With what we currently have it will be essentially (value of `operationName`):`writeCurrentBatchrum-pending-v2` or `writeCurrentBatchrum-v2`.

Assuming that tracking consent information is probably useless, we are interested mostly in the feature name, which is a part of `MetricsDispatcher` already, so can be removed from here and be an additional attribute of all the events sent. But this performance metric is not using `MetricsDispatcher`, but `InternalLogger`, which is confusing.

It would be much more sense to have operation name = `writeCurrentBatch` and feature name attached as an attribute.

Reply:
That's the thing I asked on the original thread in Slack but no - one answered so I said to go with this and discuss it here maybe.

Reply:
maybe we could just use feature name here....it will remove a lot of boiler code, what do you all think ? The only thing is that we will miss the folder type ? Granted/Pending. For that we can also pass the tracking consent in the operationName.

Reply:
I would avoid growing constructor much. Also what is the point of having tracking consent here?

Reply:
Maybe add enrich `MetricsDispatcher`? Implementation already has everything needed: logger and feature name,

Reply:
Waiting for more opinions here before getting into it but I think it is the right call instead of using the root dir name.

Reply:
the only thing is to not affect the metric which is based on the `operationName` in the dashboards

Reply:
It is our internal metric, so we can break it at any time.